src/api/spec/controllers/webui/cloud/azure/configurations_controller_spec.rb

```diff
@@ -8,24 +8,66 @@
 
   describe 'GET #show' do
     context 'without Azure configuration' do
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          get :show
+        end
+      end
+
       it 'creates an Azure configuration' do
+        expect(assigns(:azure_configuration)).not_to be_nil
```

Comment:
Something like ` expect(assigns(:azure_configuration)).to exist ` would be more speaking. I would try to avoid double negations

Reply:
@bgeuken 

> Something like `expect(assigns(:azure_configuration)).to exist` would be more speaking. I would try to avoid double negations

Agree. I will submit another pull request of the other 34 places `.not_to be_nil` appear in our test suite.